source/model-generator/TaintConfigTemplate.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <mariana-trench/model-generator/ModelTemplates.h>
+#include <mariana-trench/model-generator/TaintConfigTemplate.h>
+
+namespace marianatrench {
+
+TaintConfigTemplate TaintConfigTemplate::from_json(const Json::Value& value, Context& context) {
+  TaintConfig taint_config = TaintConfig::from_json(value, context);
+  std::vector<AnnotationFeatureTemplate> annotation_features;
+  if (value.isMember("via_annotation")) {
+    for (const auto& feature_value : JsonValidation::null_or_array(value["via_annotation"])) {
```

Comment:
nit:
`    for (const auto& feature_value : JsonValidation::null_or_array(value, "via_annotation")) {`